data/distro.yml

```diff
@@ -794,3 +794,33 @@
         <p>Note: graphical installation of Flatpak apps may not be possible with GNU Guix.</p>
       </li>
     </ol>
+    
+    
+- name: "PCLinuxOS"
+  logo: "pclinuxos.svg"
+  info: >
+    <ol class="distrotut">
+      <li>
+        <h2>Install Flatpak</h2>
+        <p>The flatpak package is available in the PCLinuxOS repositories, it can be installed using Synaptic Package Manager or by running the following in the terminal:</p>
+        <pre><code>
+          <span class="unselectable">$</span> su {Enter the root password}
```

Comment:
```suggestion
          <span class="unselectable">$</span> su
```

Reply:
By the way, sudo is not available on PCLinuxOS?

Reply:
nope:
> https://www.pclinuxos.com/forum/index.php?topic=147177.0

Reply:
Hmm, I wonder whether they have Polkit working. If not, running `flatpak remote-add` (without `--user`) will fail.